.github/ISSUE_TEMPLATE/Custom.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Meta Issues
+about: This issue type is for major meta-issues or discussions

```

Comment:
Should this filename be named "Meta.md"?

Reply:
I recognize this issue might be self-explanatory, but I'm assuming some guidance on stating that this should reference two or more child issues with perhaps something like

* [ ] Child issue 1 - Link to issue
* [ ] Child issue 2 - Link to issue
* [ ] Child issue 3 - Link to issue

Reply:
Renamed to Meta, good call. Right now, you're the meta-master, so will leave this free-form for now.